src/Models/Timeline.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Plank\Checkpoint\Models;
+
+use Illuminate\Database\Eloquent\Collection;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\HasMany;
+
+/**
+ * @property-read Collection|Checkpoint[] $checkpoints
+ */
+class Timeline extends Model
+{
+    /**
```

Comment:
I believe timelines are missing a `$fillable` property? unless i'm missing some other avenue to create timelines?

Something like this?
 ```suggestion
class Timeline extends Model
{

    protected $fillable = ['title']
    
    /**
```